platform/podjs/rollup.config.js

```diff
@@ -23,7 +23,13 @@ export default [
                 transforms: ["typescript"],
             }),
         ],
-        external: ["@polypoly-eu/rdf", "@zip.js/zip.js"],
+        external: [
+            "@polypoly-eu/rdf",
+            "@zip.js/zip.js",
+            "io-ts/Decoder",
+            "fp-ts/Either",
+            "fp-ts/function",
```

Comment:
This stuff freaks me out a bit.. I mean, it works, but how I get it, here we are basically telling Rollup that it doesn't have to bother resolving any of these modules, because they will be available at runtime, i.e. included as separate scripts in some HTML file. But we don't do that!

I already don't understand why we had to include `rdf` and `zip.js` here - but why now include these _TypeScript_ modules? Why do we even need TypeScript modules at _runtime_? Can you put my mind at ease here? :D

Reply:
Well, [this is the warning we obtain if we eliminate that](https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency). Essentially, rollup does not include any external module in the bundle. We might want to include these ones, however. I'll look for another way to include them using resolve.

Reply:
The (possibly more elegant) alternative was to use resolve, which is what I have done now. Besides, now the configuration for both bundles is the same.